### PR TITLE
Support localization and add monthly report templates

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: src/components/avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                backgroundPositionX: align,
+                backgroundPositionY: 'center',
+                backgroundSize: 'contain',
+              }
+            : {}),
+        }}
+      >
+        <img
+          src={src}
+          alt=""
```
Comment (williamchong) [inline src/components/avatar.tsx]: alt could be `avatar of ${likerID}`

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/components/avatar.tsx [hunk 2]
```diff
@@ -45,7 +48,7 @@ export const Avatar = (props: AvatarProps) => {
       >
         <img
           src={src}
-          alt=""
+          alt={displayName || likerID}
```
Comment (williamchong) [inline src/components/avatar.tsx]: `LikerID` and `displayName` are both optional props, would guarding this with `''` or making `LikerID` required better?

File: src/templates/reports/monthly/writer/index.tsx [hunk 3]
```diff
@@ -237,7 +196,7 @@ export const MonthlyReportWriterTemplate = ({
           <MjmlTable cellpadding="8px">
             {topContents.map((content, i) => (
               <WriterTopContentRow
-                key={content.url}
+                key={i}
```
Comment (williamchong) [inline src/templates/reports/monthly/writer/index.tsx]: What is the problem of using `url` here?

**Review by williamchong — CHANGES_REQUESTED** [review]
Seems all field are optional field in the type definition, not sure if good pratice

**Review by williamchong — APPROVED** [review]

File: src/components/cta-sponsor-link.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+          <MjmlRaw>
+            <div
+              style={{
+                border: `${Colors.LikeCyan} 2px solid`,
+                borderRadius: 8,
+                marginTop: 8,
+                padding: 8,
+              }}
+            >
+              {`${LIKER_LAND_ROOT}/${likerID}/civic`}
```
Comment (williamchong) [inline src/components/cta-sponsor-link.tsx]: Should this be a link wrapped in utm?

File: src/components/cta-sponsor-link.tsx [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+          paddingTop={8}
+          fontSize={12}
+          color={Colors.Grey9B}
+          align="center"
+        >
+          <FormattedMessage
+            id="cta.sponsor-link.hint"
+            values={{
+              a: (text: string) => (
+                <Link href={`${LIKER_LAND_ROOT}/creators`}>{text}</Link>
```
Comment (williamchong) [inline src/components/cta-sponsor-link.tsx]: Should this be wrapped in utm?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/components/cta-sponsor-link.tsx (hunk 5 above)
Comment (nwingt) [reply src/components/cta-sponsor-link.tsx]: Yes

File: src/components/cta-sponsor-link.tsx (hunk 4 above)
Comment (nwingt) [reply src/components/cta-sponsor-link.tsx]: This is for display only, not link

**Review by williamchong — APPROVED** [review]

**Review by williamchong — APPROVED** [review]

**Pull request closed without merging**